src/components/card/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { Card as ChakraCard } from '@chakra-ui/react';
+
+export type { CardBodyProps, CardFooterProps, CardHeaderProps } from '@chakra-ui/react';
+
+const { Root, Header, Body, Footer } = ChakraCard;
+
+Root.displayName = 'Card.Root';
+Header.displayName = 'Card.Header';
+Body.displayName = 'Card.Body';
+Footer.displayName = 'Footer.Body';
+
+export const Card = {
+    Root,
+    Body,
+    Header,
+    Footer,
+};
+
+/**
+ * @deprecated since version 3.0
```

Comment:
```suggestion
 * @deprecated please use the namespace style import instead
```
We don't match chakra versions so `3.0` is not an accurate number